src/snowcli/config.py

```diff
@@ -46,15 +46,13 @@ def save(self):
 def connect_to_snowflake(connection: Optional[str] = None, **overrides):  # type: ignore
     global snowflake_connection
     cfg = AppConfig()
-    snowsql_config = SnowsqlConfig(path=cfg.config.get("snowsql_config_path"))
+    snowsql_config = ConnectionConfigs(path=cfg.config.get("snowsql_config_path"))
```

Comment:
IMO we should change variable name (snowsql_config) because we changed class name.

Reply:
changed